tna-long-form.php

```diff
@@ -0,0 +1,181 @@
+
+        <!--Get page content-->
+		<?php
+		if ( have_posts() ):
+			while ( have_posts() ):
+				the_post();
+				?>
+                <section class="cd-section"
+                         data-section-title="<?php echo sanitize_title_with_dashes( get_the_title() ); ?>"
+                         id="<?php echo sanitize_title_with_dashes( get_the_title() ); ?>">
+					<?php
+					if ( has_post_thumbnail( $post->ID ) ): ?>
+					<?php $image = wp_get_attachment_image_src( get_post_thumbnail_id( $post->ID ), 'single-post-thumbnail' ); ?>
+                    <figure>
+                        <div class="lf-image-bg-fixed-height"
+                             style="background-image: url('<?php echo make_path_relative( $image[0] ); ?>')">
+							<?php get_template_part( 'breadcrumb' ); ?>
+                        <!-- Social Media Sharing-->
+                            <div class="archive-social-media">
+                                <div id="fb-root"></div>
+                                <script>(function (d, s, id) {
+                                        var js, fjs = d.getElementsByTagName(s)[0];
+                                        if (d.getElementById(id)) return;
+                                        js = d.createElement(s);
+                                        js.id = id;
+                                        js.src = "//connect.facebook.net/en_GB/sdk.js#xfbml=1&version=v2.5";
+                                        fjs.parentNode.insertBefore(js, fjs);
+                                    }(document, 'script', 'facebook-jssdk'));</script>
+                                <div class="fb-share-button"
+                                     data-href="change to the url of the page here"
+                                     data-layout="button_count" tabindex="0"></div>
+
+                                <span tabindex="0">
+                                    <a href="https://twitter.com/share" class="twitter-share-button"
+                                       data-url="change to the url of the page here"
+                                       data-via="UKNatArchives">Tweet</a>
+                                    <script>!function (d, s, id) {
+                                           var js, fjs = d.getElementsByTagName(s)[0], p = /^http:/.test(d.location) ? 'http' : 'https';
+                                           if (!d.getElementById(id)) {
+                                               js = d.createElement(s);
+                                               js.id = id;
+                                               js.src = p + '://platform.twitter.com/widgets.js';
+                                               fjs.parentNode.insertBefore(js, fjs);
+                                           }
+                                       }(document, 'script', 'twitter-wjs');
+                                    </script>
+                                </span>
+                            </div>
```

Comment:
This has now been taken out. My feeling is it would be better to add this from the backend in wordpress, when the template is selected.